### PR TITLE
[Buttons] Break the docs into separate articles and run the readme generator.

### PR DESCRIPTION
**Pull request by jverkoey**

### CODE REVIEW

File: components/ActivityIndicator/README.md [hunk 1]
```diff
@@ -183,3 +200,49 @@ activityIndicator.cycleColors =  @[ UIColor.blueColor,
...
+
+You can theme an activity indicator with your app's color scheme using the ColorThemer extension.
+
+You must first add the Color Themer extension to your project:
+
+```bash
+pod 'MaterialComponents/ActivityIndicator+Extensions/ColorThemer'
+```
+
+Run `pod install` again.
```
Comment (mohammadcazig) [inline components/ActivityIndicator/README.md]: This feels a bit redundant and it's not consistent with other places. e.g same doc on the top just mentions how to add the pod to podfile but doesn't mention the install.

File: components/ActivityIndicator/README.md [hunk 2]
```diff
@@ -183,3 +200,49 @@ activityIndicator.cycleColors =  @[ UIColor.blueColor,
...
+#### Swift
+```swift
+// Step 1: Import the ColorThemer extension
+import MaterialComponents.MaterialActivityIndicator_ColorThemer
+
+// Step 2: Create or get a color scheme
+let colorScheme = MDCSemanticColorScheme()
+
+// Step 3: Apply the color scheme to your component
+MDCActivityIndicatorColorThemer.applySemanticColorScheme(colorScheme, to: component)
```
Comment (mohammadcazig) [inline components/ActivityIndicator/README.md]: component might not be clear here.

File: components/ActivityIndicator/README.md [hunk 3]
```diff
@@ -183,3 +200,49 @@ activityIndicator.cycleColors =  @[ UIColor.blueColor,
...
+
+```objc
+// Step 1: Import the ColorThemer extension
+#import "MaterialActivityIndicator+ColorThemer.h"
+
+// Step 2: Create or get a color scheme
+id<MDCColorScheming> colorScheme = [[MDCSemanticColorScheme alloc] init];
+
+// Step 3: Apply the color scheme to your component
+[MDCActivityIndicatorColorThemer applySemanticColorScheme:colorScheme
```
Comment (mohammadcazig) [inline components/ActivityIndicator/README.md]: component might not be clear here.

File: components/ActivityIndicator/docs/color-theming.md [hunk 4]
```diff
@@ -17,7 +8,7 @@ You must first add the Color Themer extension to your project:
 pod 'MaterialComponents/ActivityIndicator+Extensions/ColorThemer'
 ```
 
-## Example code
+Run `pod install` again.
```
Comment (mohammadcazig) [inline components/ActivityIndicator/docs/color-theming.md]: This feels a bit redundant and it's not consistent with other places. e.g same PR on the top just mentions how to add the pod to podfile but doesn't mention the install.

File: components/Buttons/README.md [hunk 5]
```diff
@@ -185,7 +198,198 @@ getter will return the current value of the property.
...
+[MDCContainedButtonThemer applyScheme:buttonScheme toButton:button];
+```
+<!--</div>-->
+
+#### Floating action buttons
+
+<img src="assets/fab.gif" alt="An animation showing a Material Design floating action button." width="99">
+
+To theme a button as a Material Design floating action button, use `MDCFloatingActionButtonThemer`
+with an `MDCRaisedButton`.
```
Comment (mohammadcazig) [inline components/Buttons/README.md]: Change MDCRaisedButton to an MDCButton. Since MDCFloatingActionButtonThemer would work with a MDCButton.

**Review by mohammadcazig — COMMENTED** [review]
couple of nits and fixes.

**Comment by romoore** [comment]
Removing review request since this is marked blocked.

File: components/ActivityIndicator/README.md (hunk 3 above)
Comment (jverkoey) [reply components/ActivityIndicator/README.md]: Done.

File: components/ActivityIndicator/README.md (hunk 2 above)
Comment (jverkoey) [reply components/ActivityIndicator/README.md]: Done.

File: components/Buttons/README.md (hunk 5 above)
Comment (jverkoey) [reply components/Buttons/README.md]: DOne.

**Comment by jverkoey** [comment]
Ready for re-review @mohammadcazig .

**Comment by jverkoey** [comment]
Force merged because this is just a docs update.

**Pull request merged**